src/OperationResponse.php

```diff
@@ -220,6 +220,25 @@ public function getError()
     }
 
     /**
+     * Get an array containing the values of 'operationReturnType' and/or 'metadataReturnType' if
+     * they are set. The array can be passed as the $options argument to the constructor when
+     * creating another OperationResponse object.
+     *
+     * @return array
+     */
+    public function getReturnTypeOptions()
+    {
+        $options = [];
+        if (isset($this->operationReturnType)) {
```

Comment:
You don't need the isset checks. The properties are declared up at the top. If they're never set, they'll just be null. I'd prefer to have the keys always present with a possible null value than to have to check if the key is there or not. :)